Source/Core/Core/HW/DSPHLE/UCodes/AXWii.cpp

```diff
@@ -214,7 +214,7 @@ void AXWiiUCode::HandleCommandList()
 
       case CMD_OUTPUT:
       case CMD_OUTPUT_DPL2:
-        volume = m_cmdlist[curr_idx++];
+        volume = m_crc == 0xd9c4bf34 ? 0x8000 : m_cmdlist[curr_idx++];
```

Comment:
Where does this value come from? Is it hardcoded in that particular µcode?

Reply:
0x8000 is full volume. I'm just copying the behavior of CMD_OUTPUT_OLD. I haven't independently verified that this is correct.

Reply:
CMD_OUTPUT in µcode 0xd9c4bf34:

![image](https://user-images.githubusercontent.com/4209061/127325333-cad1d79d-5a1e-4055-b6d8-d16b6c8c9e79.png)

Seems to match what we do in OutputSamples (the whole clamping loop is a no-op considering the volume never changes from 0x8000).